modules/conversion-center/assets/scss/widgets/contact-buttons.scss

```diff
@@ -1,13 +1,17 @@
 .e-contact-buttons {
-    --e-contact-buttons-chat-box-width: 360px;
+    --e-contact-buttons-chat-box-width: #{$contact-buttons-default-width};
```

Comment:
We usually define the default values with the `.elementor-widget-{{ widget-name }}`, so that we don't need an additional selector in the control (see below).

See e.g. https://github.com/elementor/elementor/blob/68aec3cae0189914520c82e2a08b21f7f4997c67/assets/dev/scss/frontend/widgets/rating.scss#L1

Reply:
we've been using the same prefix in all the module. I think it was because otherwise the classnames would've been too long.
I think if we want to make this change here we also need to refactor the rest of the module, but I would keep it as a separate task.